docs/index.md

```diff
@@ -1,11 +1,12 @@
 # Legacy documentation 
 
-You should upgrade to the most recent version for documentation support. For the versions that have reached their end-of-life, we provide the legacy documentation in a PDF format. Percona does not update legacy documentation.
+You should upgrade to the most recent version for documentation support. Legacy documentation includes outdated documentation and documentation for the versions that have reached their end-of-life. We provide the legacy documentation in a PDF format. Percona does not update legacy documentation.
```

Comment:
The documentation for product versions that are either outdated or have reached their end-of-life is available in a PDF format. This documentation is not updated.

Reply:
updated